apps/vanilla/src/type-guards/is-element.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Check an item is a HTMLElement or not.
+ * @param element Element need checking.
+ */
+export function isElement(element: HTMLElement | unknown): element is HTMLElement {
```

Comment:
It should be `unknown` only.
```suggestion
export function isElement(element: unknown): element is HTMLElement {
```